### PR TITLE
Second milestone Portfolio project: mobile website section

### PR DESCRIPTION
**Pull request by wandergithub**
Added the middle sections.
* "Works" section 
* About myself

> *Used Flexbox and Grid to create the Language, frameworks, and skills list.
*Implemented "tool used info" of the cards/sections with the use of ul and li tags.
*Implemented the button interactions(enable,hover, disable).
*Added Pictures to the background and solved linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,15 +15,107 @@
...
+            <a href="#"><img src="sprites/Disabled.png" alt="Github icon"></a>
+            <a href="#"><img src="sprites/Disabled-1.png" alt="Linkdin icon"></a>
+            <a href="#"><img src="sprites/Enabled.png" alt="Angelist icon"></a>  
+            <a href="#"><img src="sprites/Disabled-2.png" alt="Twitter icon"></a>
+            <a href="#"><img src="sprites/Disabled-3.png" alt="Medium icon"></a>
+        </nav>
+        <div>
+            <img src="sprites/Group55.png" alt="Down Arrow"><h2>Work</h2>
+        </div>
+        <h2 class="title">MY RECENT WORK</h2>
```
Comment (BenMukebo) [inline index.html:66]: - [ ] Please use this image below compared to the Figma and make necessary changes 
- Please make some changes to the font size of these in the image below to match the design in the Figma

toitoi

#### Figma design

figtoi

File: style.css [hunk 2]
```diff
@@ -48,4 +49,151 @@ body {
 .text {
   color: #c8c8d4;
   font-size: 20px;
+  font-weight: 400;
+}
+
```
Comment (BenMukebo) [inline style.css:54]: - [ ]Please use a display grid for your main card container, it should be more flexible and allow you to implement desktop version

File: style.css [hunk 3]
```diff
@@ -48,4 +49,151 @@ body {
...
+#section-2 {
+  display: flex;
+  justify-content: center;
+}
+
```
Comment (BenMukebo) [inline style.css:59]: - [ ] Check this out, the user will not be able to see the text behind this light image if I can call it so 😄, please adjust it a bit to make it match the original design
- [ ] Your see project button has not the proper width, please fix it.

see

see1

- it should be like this instead,
zero

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi *** @wandergithub ***

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (wandergithub) [reply style.css:54]: @BenMukebo Sorry I don't get this part, I mean:
My card containers/pages can flex all to 1000px and beyond.

And I thought the project was about the mobile version, can you clear to me what do you mean by "allow you to implement desktop version". 

Politely:relaxed::relaxed:, why my version can't be expanded to desktop?
(Appreciate all the knowledge that you can give me with your answers, please be abundant, thanks:smile::smile:)

**Pull request merged**